decidim-api/vendor/assets/javascripts/decidim/api/graphql-docs.js

```diff
@@ -2496,308 +2496,308 @@ return /******/ (function(modules) { // webpackBootstrap
 /* 10 */
 /***/ function(module, exports) {
 
-	/*

```

Comment:
Why are there changes here?

Reply:
My mistake, I will remove these files from the pull request

Reply:
I was trying to remove these files from the PR yesterday but after pushing changes this morning I've discovered that they're still there :(. I don't know how to remove them. They shouldn't be there. The unique reason why they appear is because a git bug.